### ISSUE TITLE
Fix/Research slow /transaction.all endpoint

### ISSUE THREAD

**Issue by T-Dnzt** [omisego/elixir-omg#589]
The endpoint seems to try to load the whole table when querying a list of 200 transactions (see [explorer](http://quest.ari.omg.network/address/0x6f4a4981bc761973eb81e67731e566242c53275a) and end up crashing. I'm thinking about trying to add some indexes.

**Comment by jbunce** [comment]
Exception:

```Server: watcher.ari.omg.network:80 (http)
Request: POST /transaction.all
** (exit) exited in: GenServer.call(Sentry.TaskSupervisor, {:start_task, [{:nonode@nohost, #PID<0.16632.1255>, #PID<0.16632.1255>}, [#PID<0.16632.1255>], :monitor, {:erlang, :apply, [#Function<3.26443835/0 in Sentry.Client.do_send_event/3>, []]}], :temporary, nil}, :infinity)```

With `ERROR 53400 (configuration_limit_exceeded): temporary file size exceeds temp_file_limit (1025563kB)`. That indicates to me that the whole table is trying to be loaded into a temp file

from https://www.postgresql.org/docs/current/runtime-config-resource.html:

```19.4.2. Disk

temp_file_limit (integer)

    Specifies the maximum amount of disk space that a process can use for temporary files, such as sort and hash temporary files, or the storage file for a held cursor. A transaction attempting to exceed this limit will be canceled. The value is specified in kilobytes, and -1 (the default) means no limit. Only superusers can change this setting.

    This setting constrains the total space used at any instant by all temporary files used by a given PostgreSQL process. It should be noted that disk space used for explicit temporary tables, as opposed to temporary files used behind-the-scenes in query execution, does not count against this limit.```

**Comment by pdobacz** [comment]
fixed on `master` in #611, on `v0.1` in #596
